pitch.md

```diff
@@ -3,42 +3,65 @@
 ## Overview
 _What's this project about, briefly?_
 
+It's an RPG mapmaker language. The initial implementation will be text-based only, with a GUI added only if time permits.
+The language would let users create tile objects by supplying an image source path, specify an optional border, specify a coordinate system and a map size, and then designate rectangular areas to be filled in. Additionally, non-square "tiles" or objects can be placed on the scene by designating an "anchor point" on the input image and then specifying its coordinates on the map. Different images like this can be placed on the scene, so a "tile" may be any image, and the user will also be able to designate layering. Tiles can also be rotated and translated fairly easily. Once a program is run, an output map will be produced.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The user would be game developers (or perhaps artists).
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Well, existing map generators that I'm aware of generally have set tile sizes, which limits the ability to quickly design maps that don't look as blocky or aren't constrained by having every piece of the map fill a tile of the same size. But to have complete control over a map would require illustrating it entirely from scratch, which would take too long. This would be a tool that might keep the process quick while allowing for more artistic design decisions. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+If existing map makers count as DSLs, then these tools have already proven to address this type of need.
+Perhaps a text-based DSL isn't the most ideal way to go, but it wouldn't be a long process to make a map, and it would be easy to make minor changes without having to handle multiple layers in a software like photoshop.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+_Words cannot express my interest in game design._
+Also, map generation is one of the most basic game dev tools, and free-to-use tools can always be improved on.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain? Be careful
 to distinguish between, e.g,. a graphical interface and a linguistic interface._ 
 
+This is mostly described above. A program is just lines of code. The specifics haven't been worked out yet, but specifying tile objects might look something like:
+
+tile water1 = SRCPATH
+tile ground1 = SRCPATH {
+	edge = SRCPATH
+}
+freeform tile peninsula = SRCPATH {
+	anchor = (5,5)
+}
```

Comment:
I can't entirely understand this. I'm reading the ground and water as that the water just has a texture, and the ground has a texture but also a different one for the edges of the area that could be like a beach texture. I'm pretty unsure what the peninsula type is doing, though.